### ISSUE TITLE
Gesture to close on iOS lagging since version 2

### ISSUE THREAD

**Issue by tgensol** [react-navigation/stack#353]
**Current Behavior**

This issue appears since version 2. When using the gesture to close a view, and "quickly" trying to access the same view or another, the router will not navigate.
- If it is the same view, nothing happens,
- If it is another view, the view opens and closes

![bad with gesture](https://user-images.githubusercontent.com/7864896/72273237-49a07400-362a-11ea-8898-fd36caf35874.gif)

But, if you use the goBack button, the issue does not happen :

![good with back](https://user-images.githubusercontent.com/7864896/72273259-545b0900-362a-11ea-9070-1056ab586792.gif)

**Expected Behavior**

In the version 1, these "actions" were working just fine, and the navigation was not "lagging" when using the gesture.

**How to reproduce**

- 
**Your Environment**

| software                       | version |
| ------------------------------ | ------- |
| iOS or Android                 | .  iOS
| react-navigation               | 4.0.10
| react-navigation-stack         | 2.0.15
| react-native-gesture-handler   | 1.5.3

**Comment by appdrop** [comment]
Same for me. Works in `1.10.2`, doesn't work in `2.0.15`

**Comment by tgensol** [comment]
It is working very well with the new react-navigation 5.0.0, I think [your patch](https://github.com/react-navigation/stack/blob/master/scripts/stack.patch) is not working very well (especially I think the handleOpenRoute )

**Comment by GoktuqCan** [comment]
Also having this issue.

**Comment by Jekiwijaya** [comment]
this fixed for me:

```

const TransitionIOSSpec: TransitionSpec = {
  animation: 'spring',
  config: {
    stiffness: 1000,
    damping: 500,
    mass: 3,
    overshootClamping: true,
    restDisplacementThreshold: 40,
    restSpeedThreshold: 100
  }
};


const AppNavigator = createStackNavigator(routes, {
...
defaultNavigationOptions: {
      transitionSpec: {
        open: TransitionIOSSpec,
        close: TransitionIOSSpec
      },
}
});
```
related: https://github.com/facebook/react-native/issues/20783#issuecomment-481372109

**Comment by tgensol** [comment]
@Jekiwijaya  Not working on my side

**Comment by satya164** [comment]
Should be fixed in latest version (2.1.0). Please try and lemme know.
